go/vt/vttablet/onlineddl/executor.go

```diff
@@ -124,6 +124,7 @@ const (
 	readyToCompleteHint                      = "ready_to_complete"
 	databasePoolSize                         = 3
 	qrBufferExtraTimeout                     = 5 * time.Second
+	grpcTimeout                              = 5 * time.Second
```

Comment:
This feels a little low. We have a default topo remote operation timeout variable that is 30s. Up to you though.

Reply:
Upped to `30s`.